### ISSUE TITLE
[BUG] [Linter] Tag seems to be an orphan false positive

### ISSUE THREAD

**Issue by rafrafek** [Riverside-Healthcare/djLint#483]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
    there are some similar issues but I think this one is a little bit different
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: MacOS Ventura 13.0.1
 - Python 3.11.1
 - djLint Version 1.19.7
 - template language: --profile=jinja

## Issue and How To Reproduce

djLint gives an error:

```console
H025 2:0 Tag seems to be an orphan. <tr class="foo" foo=
```

for the template below:

```html
{% if foo.bar %}
<tr class="foo" foo="bar" bar="foo">
{% else %}
<tr>
{% endif %}
  <td>
    Foo
  </td>
</tr>
```

and `--check` tries to do:

```diff
 {% if foo.bar %}
-<tr class="foo" foo="bar" bar="foo">
-{% else %}
-<tr>
-{% endif %}
-  <td>
-    Foo
-  </td>
-</tr>
+    <tr class="foo" foo="bar" bar="foo">
+    {% else %}
+        <tr>
+        {% endif %}
+        <td>Foo</td>
+    </tr>
```

which gives me incorrect indentation:

```html
{% if foo.bar %}
    <tr class="foo" foo="bar" bar="foo">
    {% else %}
        <tr>
        {% endif %}
        <td>Foo</td>
    </tr>
```

But it is not an orphan since both `{% if %}`and `{% else %}`produce `<tr>` opening tag.

## Contents of .djlintrc/pyproject.toml [tool.djlint]
no config, running with `--profile=jinja`

**Comment by christopherpickering** [comment]
also see #88
